fundingmanager.go

```diff
@@ -176,6 +177,25 @@ type numPendingReq struct {
 	resp chan uint32
 }
 
+// TODO: (andrew.shvv) Find better place for this function.
+// checkMaxPending check number of pending channels to not exceed maximum value
+// send the lnwire.ErrorGeneric message to the offending peer if the constraint
+// is violated.
+func (f *fundingManager) checkMaxPending(peer *peer) bool {
```

Comment:
I think this function should have its scope restricted a bit. Rather than sending an error to the targeted peer if the number of pending channels exceeds the limit, only the bool value should be returned without any further action. 

So the function can mostly be reduced to (depending on the name of the method): 
```go
return len(f.activeReservations[peer.id]) >= maxActiveReservations
```